patches/typo-js+1.1.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/typo-js/typo.js b/node_modules/typo-js/typo.js
+index 68c285b..11710a1 100644
+--- a/node_modules/typo-js/typo.js
++++ b/node_modules/typo-js/typo.js
+@@ -431,7 +431,7 @@ Typo.prototype = {
```

Comment:
Does this patch work?

Reply:
nope, and it made me realized I broke the spellcheck too